server/enums/dpsRole.ts

```diff
@@ -0,0 +1,7 @@
+enum DpsRole {
```

Comment:
This enum previously existed in `roleBasedAccessControl` below 👇 
I've moved it here into the `enum` folder as that is a better / more consistent place for it (rather than it being buried in another file)